clients/drcachesim/tests/offline-phys.templatex

```diff
@@ -11,19 +11,19 @@ Adios world!
 Output format:
 <--record#-> <--instr#->: <---tid---> <record details>
 ------------------------------------------------------------
-           1           0: [ ]*T[0-9]+ <marker: version 4>
-           2           0: [ ]*T[0-9]+ <marker: filetype 0x242>
-           3           0: [ ]*T[0-9]+ <marker: cache line size [0-9]+>
-           4           0: [ ]*T[0-9]+ <marker: chunk instruction count [0-9]+>
-           5           0: [ ]*T[0-9]+ <marker: page size [0-9]+>
-           6           0: [ ]*T[0-9]+ <marker: timestamp [0-9]+>
-           7           0: [ ]*T[0-9]+ <marker: tid [0-9]+ on core [0-9]+>
-           8           0: [ ]*T[0-9]+ <marker: physical address for following virtual: 0x[0-9a-f][0-9a-f]+>
-           9           0: [ ]*T[0-9]+ <marker: virtual address for prior physical: 0x[0-9a-f][0-9a-f]+>
-          10           0: [ ]*T[0-9]+ <marker: physical address for following virtual: 0x[0-9a-f][0-9a-f]+>
-          11           0: [ ]*T[0-9]+ <marker: virtual address for prior physical: 0x[0-9a-f][0-9a-f]+>
-          12           0: [ ]*T[0-9]+ <marker: physical address for following virtual: 0x[0-9a-f][0-9a-f]+>
-          13           0: [ ]*T[0-9]+ <marker: virtual address for prior physical: 0x[0-9a-f][0-9a-f]+>
-          14           0: [ ]*T[0-9]+ <marker: timestamp [0-9]+>
-          15           0: [ ]*T[0-9]+ <marker: tid [0-9]+ on core [0-9]+>
-          16           1: [ ]*T[0-9]+ ifetch .*
+           1           0: [ ]*[0-9]+ <marker: version 4>
```

Comment:
It looks like this has a single space in brackets `[ ]`?  No reason to have the brackets then and could merge with the prior space for ` +`.